src/migrations/20220103142419-remove-grantee-record-flag.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await queryInterface.bulkUpdate('Users', {
+        flags: [],
+      }, {}, { transaction });
+    },
+  ),
+  down: async (queryInterface) => queryInterface.sequelize.transaction(
```

Comment:
Is there ever going to be a case where we might want to add it back (if possible)?

Reply:
I don't see how/why at this point, maybe when we're storing more historical data that would be possible

Reply:
I guess we could always add it back manually. I was just thinking hypothetically if some issue was found and we wanted to put it back behind the feature flag.

Reply:
no that makes sense - that was kind of why I left the flag in place on the backend for a little bit, in case something came up in the first couple of weeks